yb-voyager/cmd/analyzeSchema.go

```diff
@@ -623,18 +623,71 @@ var MigrationCaveatsIssues = []string{
 	queryissue.ALTER_TABLE_ADD_PK_ON_PARTITIONED_TABLE,
 	queryissue.FOREIGN_TABLE,
 	queryissue.POLICY_WITH_ROLES,
-	queryissue.UNSUPPORTED_DATATYPE_LIVE_MIGRATION,
-	queryissue.UNSUPPORTED_DATATYPE_LIVE_MIGRATION_WITH_FF_FB,
+}
```

Comment:
Can you put this `migrationCaveatsIssues` inside the `convertIssueInstanceToAnalyzeIssue` , to avoid any usage of it outside that function as its not complete now.